reference/7.1/Microsoft.PowerShell.Core/About/about_Operators.md

```diff
@@ -319,21 +319,29 @@ A cast can also be performed when a variable is assigned to using [cast notation
 
 #### Comma operator `,`
 
-As a binary operator, the comma creates an array. As a unary operator, the
-comma creates an array with one member. Place the comma before the member.
+As a binary operator, the comma creates an array or appends to the array being
+created. In expression mode, as a unary operator, the comma creates an array
+with just one member. Place the comma before the member.
 
 ```powershell
 $myArray = 1,2,3
 $SingleArray = ,1
+Write-Output (,1)
```

Comment:
```suggestion
$ExpressionSingleArray =  (,1)
```

Reply:
The point is, `Write-Output ,1` does not work, so you have to force expression mode here.